recipes-crosswalk/crosswalk/crosswalk_20.50.533.12.bb

```diff
@@ -419,6 +448,7 @@ DEFAULT_CONFIGURATION = "\
 
 # Not available on Yocto, so we use
 # what comes bundled with Chromium.
+# -Duse_system_flac=1
```

Comment:
Uh, I just mixed up several contexts while trying to explain the error...

In this recipe, your findings are correct.

However, strictly speaking, `PKG_CONFIG_PATH` and its friends need to be unset/reset to match the environment expectation of GYP build, in which case these environment variables should just tell `pkg-config` where the sysroot for build host is. The expectation is not clearly documented in Chromium, but it can be concluded from a few commits (e.g. https://chromium.googlesource.com/chromium/src/+/f83a08c9c9156960d505472108a5c96ee2972820) that developers (BitBake here) should not worry about these environment variables. The search path of `.pc` files for target is derived from the GYP variable `sysroot`. It may still work by leaving these environment variables set by BitBake unchanged, but it's risky because `pkg-config` will use `.pc` files in the target sysroot while building tools (if any) that run on the build host.

I tried to make some changes (vaguely mentioned as a TODO about incorrect toolchain search paths below) to these environment variables some time ago so that the environment can match GYP's expectation, and then met the same error. This time the error is caused by what I've said earlier today. Sorry for the confusion.